scheduler/app/libs/tasks/__init__.py

```diff
@@ -9,3 +9,4 @@
 from ..pd_inter_calc import portfolio_midnight_filler
 
 from ..types import Treasury
+from .get_assets import *
```

Comment:
Let's try to always import explicitely: https://stackoverflow.com/questions/2386714/why-is-import-bad